examples/guided.py

```diff
@@ -97,33 +121,45 @@ def ask_user_questions():
 
 					# Get a valid & supported filesystem for the partition:
 					while 1:
-						new_filesystem = input(f"Enter a valid filesystem for {partition} (leave blank for {partition.filesystem}): ").strip(' ')
+						new_filesystem = input(
+							f"Enter a valid filesystem for {partition} (leave blank for {partition.filesystem}): "
+						).strip(' ')
 						if len(new_filesystem) <= 0:
 							if partition.encrypted and partition.filesystem == 'crypto_LUKS':
 								old_password = archinstall.arguments.get('!encryption-password', None)
 								if not old_password:
 									old_password = input(f'Enter the old encryption password for {partition}: ')
 
-								if (autodetected_filesystem := partition.detect_inner_filesystem(old_password)):
+								if (
+									autodetected_filesystem := partition.detect_inner_filesystem(old_password)
+								) :
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability. Absolutely no!